--- a/DEPS	2019-02-20 16:29:30.095863660 -0500
+++ b/DEPS	2019-02-20 16:31:16.989165937 -0500
@@ -103,7 +103,7 @@
   'angle_root': 'src/third_party/angle',
 
   # luci-go CIPD package version.
-  'luci_go': 'git_revision:25958d48e89e980e2a97daeddc977fb5e2e1fb8c',
+  'luci_go': 'git_revision:ed999c8ac210539ae9d71f65436a2c47015712d5',
 
   'android_git': 'https://android.googlesource.com',
   'aomedia_git': 'https://aomedia.googlesource.com',
